src/bokeh/server/views/autoload_js_handler.py

```diff
@@ -54,11 +54,14 @@ class AutoloadJsHandler(SessionHandler):
     '''
 
     def set_default_headers(self):
-        self.set_header("Access-Control-Allow-Headers", "*")
-        self.set_header('Access-Control-Allow-Methods', 'PUT, GET, OPTIONS')
```

Comment:
This header is only relevant for an `OPTIONS` request. I moved it into the specific method.